### DISCUSSION TITLE
Z Offset Menu Homing

### DISCUSSION THREAD

**Discussion by skaza2125** [Jyers/Marlin#856] [Ideas]
I think it Would be a good idea to make the Homing option in the Z-offset menu a home all, and not just a home Z. Maybe it's just me but I tried to adjust the z offset and being stupid I forgot to home it, causing some grinding on my motors. It was definitely my fault, but I think having a home all option there would prevent other people from having a similar issue? lemme know what you think!

**Comment by Jyers** [comment]
Interesting, home z should force the x and y axis to home if they weren't already. I'll take a look to see what might cause that.
